Fail to builde mymodule in master branch

**Describe the bug**
Build mymodule in master branch leads to the following error:

> examples/MyModule/recording_backend_soundclick.h:31:10: fatal error: SFML/Audio.hpp: No such file or directory

**To Reproduce**
Steps to reproduce the behavior:
1. Install nest master branch without external module
```
$ mkdir nest-master-make-dir
$ cd nest-master-make-dir
$ cmake -DCMAKE_INSTALL_PREFIX:PATH=<NEST_INSTALL_DIR> -Dwith-mpi=ON -Dwith-python=3 -DPYTHON_EXECUTABLE=`which python3.6` -DNOSETESTS=`which nosetests` -DCYTHON_EXECUTABLE=`which cython` -Dwith-boost=ON -Dwith-includes=<BOOST_INCLUDE_PATH> ../nest-simulator-master
$ source <NEST_INSTALL_DIR>/bin/nest_vars.sh
```
2. Build example mymodule
```
$ cd ..
$ mkdir mymodule-master-make-dir
$ cd mymodule-master-make-dir
$ cmake ./nest-simualtor-master/example/MyModule
$ make
```
3. Output example
```
In file included from /ANYDIR/nest-simulator-master/examples/MyModule/mymodule.cpp:32:
/ANYDIR//nest-simulator-master/examples/MyModule/recording_backend_soundclick.h:31:10: fatal error: SFML/Audio.hpp: No such file or directory
 #include <SFML/Audio.hpp>
```

**Expected behavior**
Successfull build

**Desktop/Environment (please complete the following information):**
 - OS: CentOS 7.7.1908
 - Browser: [if applicable, e.g. chrome, safari]
 - Shell: bash
 - Python-Version: Python 3.6
 - NEST-Version: nest git hash: d3d110f
 - Installation: compiled from source

**Additional context**
The following change in mymodule.cpp solves the issue:
```
--- a/examples/MyModule/mymodule.cpp
+++ b/examples/MyModule/mymodule.cpp
@@ -29,7 +29,9 @@
 #include "drop_odd_spike_connection.h"
 #include "pif_psc_alpha.h"
 #include "step_pattern_builder.h"
+#ifdef HAVE_SFML_AUDIO
 #include "recording_backend_soundclick.h"
+#endif
 #include "recording_backend_socket.h"
 
 // Includes from nestkernel:
```

--- a/examples/MyModule/mymodule.cpp
+++ b/examples/MyModule/mymodule.cpp
@@ -29,7 +29,9 @@
 #include "drop_odd_spike_connection.h"
 #include "pif_psc_alpha.h"
 #include "step_pattern_builder.h"
+#ifdef HAVE_SFML_AUDIO
 #include "recording_backend_soundclick.h"
+#endif
 #include "recording_backend_socket.h"
 
 // Includes from nestkernel:

